runtime/doc/lua.txt

```diff
@@ -2818,20 +2820,20 @@ vim.keymap.set({mode}, {lhs}, {rhs}, {opts})                *vim.keymap.set()*
 <
 
     Parameters: ~
-      • {mode}  (`string|table`) Mode short-name, see |nvim_set_keymap()|. Can
-                also be list of modes to create mapping on multiple modes.
+      • {mode}  (`string|string[]`) Mode short-name, see |nvim_set_keymap()|.
+                Can also be list of modes to create mapping on multiple modes.
       • {lhs}   (`string`) Left-hand side |{lhs}| of the mapping.
       • {rhs}   (`string|function`) Right-hand side |{rhs}| of the mapping,
                 can be a Lua function.
-      • {opts}  (`table?`) Table of |:map-arguments|.
-                • Same as |nvim_set_keymap()| {opts}, except:
-                  • "replace_keycodes" defaults to `true` if "expr" is `true`.
-                  • "noremap": inverse of "remap" (see below).
-                • Also accepts:
-                  • "buffer": (integer|boolean) Creates buffer-local mapping,
-                    `0` or `true` for current buffer.
-                  • "remap": (boolean) Make the mapping recursive. Inverse of
-                    "noremap". Defaults to `false`.
+      • {opts}  (`table?`) Table of |:map-arguments|. Same as
+                |nvim_set_keymap()| {opts}, except:
+                • {replace_keycodes} defaults to `true` if "expr" is `true`.
+
```

Comment:
Should we still mention that `remap` is disabled by default, unlike `nvim_set_keymap` which enables it by default.